cmd/minikube/cmd/root.go

```diff
@@ -73,15 +74,24 @@ var RootCmd = &cobra.Command{
 			}
 		}
 
-		shouldShowLibmachineLogs := viper.GetBool(showLibmachineLogs)
-		if glog.V(3) {
-			log.SetDebug(true)
+		if viper.GetBool(showLibmachineLogs) {
+			fmt.Println(`
+--show-libmachine-logs is deprecated.
```

Comment:
Yeah, it probably should work as intended while its deprecated.  I will change that